specification/visualstudio/resource-manager/Microsoft.VisualStudio/preview/2018-08-01-preview/Projects.json

```diff
@@ -0,0 +1,717 @@
+    "title": "Visual Studio Projects Resource Provider Client",
+    "version": "2018-08-01-preview",
+    "description": "Use these APIs to manage Visual Studio Team Services resources through the Azure Resource Manager. All task operations conform to the HTTP/1.1 protocol specification and each operation returns an x-ms-request-id header that can be used to obtain information about the request. You must make sure that requests made to these resources are secure. For more information, see https://docs.microsoft.com/en-us/rest/api/index."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "flow": "implicit",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/microsoft.visualstudio/account/{rootResourceName}/project": {
+      "get": {
+        "tags": [
+          "Projects"
+        ],
+        "summary": "Projects_ListByAccountResource",
+        "description": "Gets all Visual Studio Team Services project resources created in the specified Team Services account.",
+        "operationId": "Projects_ListByAccountResource",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "in": "path",
+            "name": "rootResourceName",
+            "description": "Name of the Team Services account.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The operation succeeded. The response contains the details of the Visual Studio Team Services project resources created in the specified Team Services account.",
+            "schema": {
+              "$ref": "#/definitions/ProjectResourceListResult"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get a list of project resources in the Team Services account": {
+            "$ref": "./examples/GetProjectResources_List.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/microsoft.visualstudio/account/{rootResourceName}/project/{resourceName}": {
+      "put": {
+        "tags": [
+          "Projects"
+        ],
+        "summary": "Projects_Create",
```

Comment:
Given PUT semantics, the standard is to call these `CreateOrUpdate` instead of just `Create`

Reply:
changed to Projects_CreateOrUpdate